contributors/design-proposals/kubectl-create-configmap-env-file.md

```diff
@@ -0,0 +1,55 @@
+# Kubectl create configmap --env-file 
```

Comment:
I will have to think about that. The env file is currently string=string and secrets are more string=[]byte. I don't want to get into supporting BASE64 encodings or ways to really bring in binary data since it is not currently specified.
If we are just talking about string=string, fine which will mean passwords but not keys.